packages/editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@joincivil/editor",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
Please don't update versions manually. Every once in a while (sprint?) I do a release of all packages through `lerna` that tags the commit all in one place.

If you need to upgrade this package, use lerna which add's tags etc.